### ISSUE TITLE
[Improvement] add validate check for ColumnPosition when updating column positions.

### ISSUE THREAD

**Issue by FANNG1** [apache/gravitino#4412]
### What would you like to be improved?

It seems ridiculous to update the column position to `default`.  I think we should add a validate check when constructing `UpdateColumnPosition`

**Comment by FANNG1** [comment]
cc @mchades @caican00 , WDYT?

**Comment by caican00** [comment]
> cc @mchades @caican00 , WDYT?

yeah, if we want to place the column to the end, it seems batter to use `after`.
And `default` position is better used in `AddColumn`.

**Comment by LiuQhahah** [comment]
Hi @FANNG1 
Please assign to me .
Thanks.
